ByBit.Net/Clients/SpotApi/v3/BybitClientSpotApiExchangeDataV3.cs

```diff
@@ -41,9 +41,10 @@ public async Task<WebCallResult<DateTime>> GetServerTimeAsync(CancellationToken
         #region Get symbols
 
         /// <inheritdoc />
-        public async Task<WebCallResult<BybitSpotSymbolWrapper>> GetSymbolsAsync(CancellationToken ct = default)
+        public async Task<WebCallResult<IEnumerable<BybitSpotSymbolV3>>> GetSymbolsAsync(CancellationToken ct = default)
         {
-            return await _baseClient.SendRequestAsync<BybitSpotSymbolWrapper>(_baseClient.GetUrl("spot/v3/public/symbols"), HttpMethod.Get, ct, null).ConfigureAwait(false);
+            var result = await _baseClient.SendRequestAsync<BybitSpotSymbolWrapper>(_baseClient.GetUrl("spot/v3/public/symbols"), HttpMethod.Get, ct, null).ConfigureAwait(false);
+            return result.As(result.Data.Symbols);
```

Comment:
If  the call fails result.Data will be null, so unfortunately this will throw an exception then. You'll need a null check, `result.Data?.Symbols`.

Reply:
Agree, tnx.